lib/src/folding/parsers/fallback.dart

```diff
@@ -1,33 +1,41 @@
-import 'dart:math';
+// ignore_for_file: use_string_buffers
 
 import 'package:charcode/ascii.dart';
-import 'package:collection/collection.dart';
 import 'package:highlight/highlight_core.dart';
+import 'package:tuple/tuple.dart';
 
 import '../../code/code_lines.dart';
 import '../foldable_block_type.dart';
 import 'text.dart';
 
 /// A parser for foldable blocks from raw text.
 class FallbackFoldableBlockParser extends TextFoldableBlockParser {
-  final List<String> singleLineCommentSequences;
   final List<String> importPrefixes;
 
-  /// The size of a rolling window to remember processed characters.
-  final int _tailLength;
+  /// [ ['/*', '*/'] , ...]
+  final List<Tuple2<String, String>> multilineCommentSequences;
+
+  /// ['//', '#']
+  final List<String> singleLineCommentSequences;
+
+  String? _startedMultilineCommentWith;
```

Comment:
Maybe rename if no line breaks are caused?
```suggestion
  String? _startedMultilineCommentSequence;
```